packages/mimir/lib/src/impl/ffi/io.dart

```diff
@@ -1,18 +1,16 @@
 import 'dart:ffi';
 import 'dart:io';
 
+import 'package:flutter_rust_bridge/flutter_rust_bridge_for_generated.dart';
 import 'package:meta/meta.dart';
-import 'package:mimir/src/bridge_generated.dart';
-
-/// Represents the external library for mimir
-///
-/// Will be a DynamicLibrary for dart:io or WasmModule for dart:html
-typedef ExternalLibrary = DynamicLibrary;
+import 'package:mimir/src/frb_generated.dart';
 
 @internal
 // ignore: public_member_api_docs
-EmbeddedMilli createWrapperImpl(ExternalLibrary dylib) =>
-    EmbeddedMilliImpl(dylib);
+Future<RustLibApi> createWrapperImpl(String dylibPath) async {
```

Comment:
Using `dylibPath` will break Apple device builds since they use static linking in `flutter_mimir`. Would you mind reverting all of the `dylibPath` related-changes (I'm sorry, I imagine that was a ton of work) to instead use whatever the proper type is nowadays? Based on my first look of things, that types seems like it'd be the return result of `ExternalLibrary.open()`

Reply:
I don't see an alternative to passing in a dynamic library path https://github.com/fzyzcjy/flutter_rust_bridge/blob/9e5c6e54723700e8436bc76016173e0391befa8e/frb_dart/lib/src/platform_types/_io.dart#L40

Reply:
Hmm, for static linking it looks like we'd need [`ExternalLibrary.process`](https://github.com/fzyzcjy/flutter_rust_bridge/blob/9e5c6e54723700e8436bc76016173e0391befa8e/frb_dart/lib/src/platform_types/_io.dart#L62), but that seems to be strongly discouraged in FRB v2. Guess that means Mimir will need to switch to dynamic linking/loading on iOS/macOS (I think all other platforms use dynamic loading already); would you be willing to look into that switch as a part of this PR?

Also, instead of `ExternalLibrary.open` (which is documented as internal), it looks like we should be using [this function](https://pub.dev/documentation/flutter_rust_bridge/latest/flutter_rust_bridge/loadExternalLibrary.html). Would you mind switching to that?

Adding onto that last paragraph, if Mimir ever got web support, we wouldn't be able to pass around a `dylibPath`. It looks like the proper solution here is to pass around an [`ExternalLibraryLoaderConfig`](https://pub.dev/documentation/flutter_rust_bridge/latest/flutter_rust_bridge_for_generated_web/ExternalLibraryLoaderConfig-class.html) object, especially since `ExternalLibrary` itself is marked as internal (which feels like it's pushing the limit on what you can mark as "internal" and get away with, but alas...).

Reply:
I don't have an iphone or mac to test on, my use case is for android and linux.

Good call, I changed it to use `ExternalLibraryLoaderConfig`.